src/vs/workbench/parts/terminal/electron-browser/terminal.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import * as os from 'os';
+import platform = require('vs/base/common/platform');
+import processes = require('vs/base/node/processes');
+
+const powershellPath = `${ process.env.SystemRoot }/system32/WindowsPowerShell/v1.0/powershell.exe`;
```

Comment:
That is correct.  Since VS Code is a 32-bit process, the 64-bit version of PowerShell has to be explicitly launched otherwise you'll get the 32-bit version.  Here's the relevant code from the PowerShell extension:

https://github.com/PowerShell/vscode-powershell/blob/5b32676e9559233f8759e3b0c60583c60e70eb94/src/session.ts#L493